### ISSUE TITLE
Error checking for new version

### ISSUE THREAD

**Issue by fairwood136** [containous/traefik#5256]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?
Started up a docker version with the lastest release candidate
-->

### What did you expect to see?
Nothing

### What did you see instead?

```
shared/traefik.log:time="2019-08-27T14:32:48Z" level=warning msg="Error checking new version: BaseURL must have a trailing slash, but \"https://update.traefik.io\" does not"
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Docker
Version:      2.0.0-rc1
Codename:     montdor
Go version:   go1.13rc1
Built:        2019-08-26T17:47:29Z
OS/Arch:      linux/amd64
```

**Comment by dduportal** [comment]
Hi @fairwood136 , thanks for your interest and testing!

Can you elaborate on the command you used to meet this behavior?

With this command, we cannot reproduce it:

```shell
$ docker run --rm -d --name edge-lb traefik:v2.0.0-rc1
...
$ docker logs edge-lkb
time="2019-08-27T15:14:12Z" level=error msg="\nYou haven't specified the sendAnonymousUsage option, it will be enabled by default.\n"
```

**Comment by traefiker** [comment]
Closed by #5266.

**Comment by dduportal** [comment]
@fairwood136 It was a real issue, and the team fixed this (thanks!).

For later readers: you have to wait ~15 min to be sure to reproduce the issue.
